compliance/azure/instances_without_fnm_agent/README.md

```diff
@@ -0,0 +1,93 @@
+## Checks Azure instances that are not running the FlexNet Inventory Agent
```

Comment:
should change to the new proposed name: Azure Instances not running FNMS agent

Reply:
Changed.